### PR TITLE
Updates for @textile/go-mobile@1.0.0-rc39

### PR DESCRIPTION
**Pull request by sanderpick**

### CODE REVIEW

**Comment by sanderpick** [comment]
Ok. Good to go here on my end.

File: lib/Textile/API/index.ts [hunk 1]
```diff
@@ -277,15 +278,15 @@ class API {
...
           const queryEvent = QueryEvent.decode(buffer)
           switch (queryEvent.type) {
-            case 1:
-              cleanup()
-              resolve()
-              break
-            case 0:
+            case QueryEvent.Type.DATA:
               if (queryEvent.data) {
                 handler(queryEvent.data as Contact)
```
Comment (asutula) [inline lib/Textile/API/index.ts]: I know this line isn't part of your PR, but this is wrong. Need to dig down into `queryEvent.data.value` and use `Contact.decode` or whatever.

Comment (sanderpick) [reply lib/Textile/API/index.ts]: Sure, will fix

File: lib/Textile/API/index.ts [hunk 2]
```diff
@@ -339,19 +340,19 @@ class API {
     await TextileNode.stop()
   }
 
-  threadFeed = async (offset: string, limit: number, threadId?: string): Promise<ReadonlyArray<ThreadFeedItem>> => {
-    const result = await TextileNode.threadFeed(offset, limit, threadId)
-    return JSON.parse(result) as ReadonlyArray<ThreadFeedItem>
+  feed = async (offset: string, limit: number, mode: FeedMode, threadId?: string): Promise<IFeedItemList> => {
+    const result = await TextileNode.feed(offset, limit, mode, threadId)
+    return JSON.parse(result) as IFeedItemList
```
[the remark this answers is not in the document]
Comment (sanderpick) [reply lib/Textile/API/index.ts]: I did yeah, will update

[the remark this answers is not in the document]
Comment (sanderpick) [reply lib/Textile/API/index.ts]: done

File: lib/Textile/API/index.ts (hunk 1 above)
Comment (sanderpick) [reply lib/Textile/API/index.ts]: fixed, added `local` to the handler because this might be useful (local mean the contact already exists)

**Comment by sanderpick** [comment]
Ah, good catch. I preemptively remove the hardcoded `File` type and switched to the pb version FileIndex, but actually, since that object has a special field, `IStruct`, that's gonna break. Need to put `File` back. What were the other similar changes you saw?

**Comment by asutula** [comment]
I guess I exaggerated... I saw that one, but stopped looking closely after that :) Will look closely now.

**Comment by asutula** [comment]
Ok just that one instance.

**Comment by sanderpick** [comment]
Cool, got trigger happy deleting types. Thanks for the looks... so easy to miss stuff in the big updates.

**Pull request merged**